packages/bsky/src/api/app/bsky/graph/getList.ts

```diff
@@ -88,6 +114,31 @@ const presentation = (
   return { list, items, cursor }
 }
 
+const maybeGetBlocksForReferenceList = async (input: {
```

Comment:
We may need similar logic in `app.bsky.graph.getStarterPack` for `starterPack.listItemsSample`.

Reply:
Added something for that, tell me if it makes sense or not - seemed to me to make the most sense to handle it inside of `hydrateStarterPacks()`.